app/screens/index.ts

```diff
@@ -3,3 +3,4 @@ export * from './Initial';
 export * from './Menu';
 export * from './Onboarding';
 export * from './Reanimated';
+export * from './reusableScreens';
```

Comment:
Lower valiue?

Reply:
Please, make it work with uppercase..

Reply:
It may be really difficult to change it for GitHub correctly after your commits, pay maximum attention to this